sdk/digitaltwins/azure-digitaltwins-core/src/main/java/com/azure/digitaltwins/core/DeleteRelationshipRequestOptions.java

```diff
@@ -0,0 +1,21 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.digitaltwins.core;
+
+
+import java.util.List;
+
+/**
+ * Optional settings that are specific to calls to {@link DigitalTwinsClient#updateComponent(String, String, List)} and its overloads.
+ */
+public class DeleteRelationshipRequestOptions extends RequestOptions {
+    // This class exists to be added to later if the deleteRelationship APIs get a new optional parameter in later service
+    // API versions and so that we don't have to expose that new optional parameter for other APIs like deleteDigitalTwin
+
+    @Override
+    public DeleteRelationshipRequestOptions setIfMatch(String ifMatch) {
```

Comment:
Ah, is this because of the need for fluent getters and setters?

Reply:
Yeah, without the override, the fluent return value will be of type RequestOption, not the inheriting class

Reply:
It's a bit of code duplication, but at least the javadocs are inherited correctly